DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Speckle.Connectors.RevitShared.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.Build.NoTargets/3.3.0">
+  <PropertyGroup>
+  <TargetFramework>netstandard1.0</TargetFramework>
+  <IsPackable>false</IsPackable>
+  </PropertyGroup>
+  
+
+  <ItemGroup>
+    <Compile Include="**\*.cs" Exclude="obj\**\*.*"/>
+  </ItemGroup>
+  <ItemGroup>
+    <Page Include="**\*.xaml"  Exclude="obj\**\*.*">
+      <SubType>Designer</SubType>
+      <Generator>MSBuild:Compile</Generator>
+    </Page>
```

Comment:
If the .xaml and .cs files live in this project, why do we need to also add them to the specific 2023 project as well?

Also, that exclude for the `obj` folder feels wrong 😅

Reply:
they don't really "live" here....it's not used by other csprojs if you look